packages/package-json-lint/src/config/index.js

```diff
@@ -0,0 +1,15 @@
+const getConfig = async () => ({
```

Comment:
I assume the async here is future thinking?

Reply:
Yep. Eventually I'll need to do some file analysis.